lib/buffer.js

```diff
@@ -3,12 +3,12 @@
 
 const binding = process.binding('buffer');
 const { isArrayBuffer } = process.binding('util');
+const { deprecate } = require('internal/util');
 const bindingObj = {};
 const internalUtil = require('internal/util');
 
-class FastBuffer extends Uint8Array {}
+const FastBuffer = (() => class Buffer extends Uint8Array {})();
```

Comment:
@ChALkeR it was necessary because the `FastBuffer.prototype.constructor = Buffer;` line was removed. It was removed to prevent Uint8Array method from being affected by the deprecation.